src/components/ScriptExplorer/ScriptEntry.jsx

```diff
@@ -250,23 +246,12 @@ class ScriptEntry extends React.Component {
 
   fetchUTXOs = async (multisig) => {
     const {network, client} = this.props;
-    let utxos = null;
-    try {
-      utxos = await fetchAddressUTXOs(multisig.address, network, client);
```

Comment:
Here is one of the places where the code after this was folded into `fetchAddressUTXOs`